src/bptl/tasks/api.py

```diff
@@ -29,14 +31,16 @@ class TaskPerformed(Exception):
     pass
 
 
-def execute(task: ExternalTask, registry: WorkUnitRegistry = register) -> None:
+def execute(
+    task: Union[ExternalTask, ServiceTask], registry: WorkUnitRegistry = register
```

Comment:
And use the abstract class here instead of the concrete classes for all BPMN brands.

Reply:
done